tests/src/JIT/HardwareIntrinsics/X86/Avx2/GatherMaskVector128_r.csproj

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please add these as templated tests, since we are trying to reduce the number of manually managed tests that we have for the intrinsics.

Reply:
and when doing so, please ensure that the new templates follow the same general pattern as previous templates, to ensure that all the special-cases are covered (such as indirect invocation)

Reply:
Gather intrinsic have very special overloads and semantics that do not work with the current test template framework.

Let me move the test cases later in a new PR, some applications are waiting for these intrinsic

Reply:
Then, at the very least, you need to ensure all the special cases are still covered in the manual code.

The current templates cover 14 or so scenarios that are meant to test the various code patterns that must be supported and that might influence the JIT (including things like `indirect invocation`, various patterns that may impact `containment`, etc).

Reply:
I'm also not sure I agree with "does not work with the test template framework".

The framework is very generic and just works with a template and named string values. You should be able to create a template that fits your needs (and that follows the existing pattern) and have it just work.